tests/figs/jasp-deps.txt

```diff
@@ -0,0 +1 @@
+- jaspGraphs: 0.5
```

Comment:
does this mean that if I bump the jaspGraphs version I have to increment this file in every module?

Reply:
Yes. We can't guarantee the jaspGraphs version developers use in new modules, so we can't do this centrally.
Besides all plots need to be verified after a jaspGraphs bump anyhow.